test/functional/cypress/specs/events/filter-spec.js

```diff
@@ -558,6 +558,46 @@ describe('events Collections Filter', () => {
         cy.get(latestStartElement).clear()
       })
     })
+
+    context('Aventri ID', () => {
+      const element = '[data-test="aventri-id-filter"]'
+      const aventriId = '200300400'
+      const invalidAventriId = 'Testing'
+      const invalidAventriIdNumbers = '200300400500600'
+      const queryParamWithAventriId = 'aventri_id=200300400'
+
+      context('should filter from user input', () => {
+        it('should aventri ID from user input to query param', () => {
+          cy.get(element).type(`${aventriId}{enter}`)
+          cy.url().should('include', queryParamWithAventriId)
+        })
+
+        it('should not add anything to the query param if the name is backspaced', () => {
+          cy.get(element).type(`{selectAll}{backspace}{enter}`)
+          cy.url().should('not.include', queryParamWithAventriId)
+        })
+
+        it('should not allow non numerical characters', () => {
+          cy.get(element).type(`${invalidAventriId}{enter}`)
+        })
+
+        it('should allow 9 digits', () => {
```

Comment:
Maybe the following test message would be a bit more descriptive? :) 
```suggestion
        it('should truncate any long numbers to 9 digits', () => {
```